packages/webpack/package.json

```diff
@@ -38,7 +38,7 @@
     "copy-webpack-plugin": "^10.2.4",
     "css-loader": "^6.4.0",
     "css-minimizer-webpack-plugin": "^5.0.0",
-    "dotenv": "~16.3.1",
```

Comment:
I thought we still need dotenv

Reply:
create a separate pr for this, no loner needed in the package.json